### PR TITLE
QOLOE-125 Add head meta elements which will include replaced version details in dist,

### PR DESCRIPTION
**Pull request by duttonw**
Also rejigged components to group like components together to make it easier to find things.

Still requires cdn deployments to kick off as the final piece of the puzzle.

### CODE REVIEW

File: src/components/bs5/head/head.stories.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      control: { type: 'radio' ,
+        labels: { 'DEV': 'DEV', 'TEST' : 'TEST','BETA':'BETA','STAGING':'STAGING','PROD':'PROD','/__data/assets/git_bridge/0026/471752/': 'SQUIZ Custom'},
+      },
+      options: [
+        "DEV",
+        "TEST",
+        "BETA",
+        "STAGING",
+        "PROD",
+        "/__data/assets/git_bridge/0026/471752/",
```
Comment (jortronQGov) [inline src/components/bs5/head/head.stories.js:24]: The trailing slash on this Squiz custom option (or the leading slash in the handlebars template) is causing a double up in the output
image

File: src/components/bs5/head/Head.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import Component from '../../../js/QGDSComponent.js'
+import template from "./head.hbs?raw";
+
+export class Head {
+
+  // Use the global Component class to create a new instance of the Loading Quickexit component.
```
Comment (jortronQGov) [inline src/components/bs5/head/Head.js:6]: Update reference to Quickexit component

File: src/components/bs5/head/head.stories.js (hunk 1 above)
Comment (duttonw) [reply src/components/bs5/head/head.stories.js]: fixed the story, it shows how a custom text can be used.

File: src/components/bs5/head/head.data.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "cdn": "{DEV|TEST|BETA|STAGING|PROD|/__data/assets/git_bridge/0026/471752/}"
```
Comment (jortronQGov) [inline src/components/bs5/head/head.data.json:2]: Trailing slash

File: src/components/bs5/head/head.stories.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  decorators:[Story => {
+    return `
+          ${Story()}
+      `;
+  }]
+};
+
+export const SQUIZ = {
+  args: {
+    cdn: "/__data/assets/git_bridge/0026/471752/"
```
Comment (jortronQGov) [inline src/components/bs5/head/head.stories.js:67]: Trailing slash

**Review by jortronQGov — APPROVED** [review]
A couple more trailing slashes that could be cleaned up for example's sake.

**Review by jortronQGov — APPROVED** [review]

**Pull request merged**